apps/web/contexts/contractsContext.tsx

```diff
@@ -39,8 +39,7 @@ export function ContractsProvider({ children }: Props) {
 
   return (
     <>
-      {/* @ts-ignore */}
-      <ContractsContext.Provider value={value}>{children}</ContractsContext.Provider>
+      <ContractsContext.Provider value={value as any}>{children}</ContractsContext.Provider>
```

Comment:
I'm not sure if we would address the `any` later on or not, because we generally  try to stay away from using `any` as it simply means we're ignoring typesafety in this case.

Reply:
Yeah, for this particular instance I have no idea why value is not assignable to Provider and why it is still not giving issues at runtime